Misc/NEWS.d/next/Library/2024-10-26-22-46-50.gh-issue-126016.gUWIIK.rst

```diff
@@ -0,0 +1 @@
+Fixed crash upon using CTRL+C while joining a thread containing a subinterpreter.
```

Comment:
Maybe NEWS is not needed. It seems that the internal state is not exposed and this should be normal behavior.

Reply:
No, bugfixes need NEWS entries. `_interpreters` isn't considered internal because it's used by Eric's PEP 734 third party.